### PR TITLE
upload zipped pack method in api client

### PR DESCRIPTION
**Pull request by ilappe**
## Status
In Progress

## Related Issues
https://github.com/demisto/etc/issues/33528

## Description
Add new method to support upload zip to marketplace 

## Must have
- [x] Changelog entry

### CODE REVIEW

File: server_api_swagger.json [hunk 1]
```diff
@@ -1755,6 +1755,33 @@
...
+      "post": {
+        "description": "Upload an pack to marketplace in Demisto",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Upload an pack as zip",
+        "operationId": "zippedPackUpload",
```
Comment (glicht) [inline server_api_swagger.json:1768]: Please use `uploadContentPacks` (same as used in server)

Comment (ilappe) [reply server_api_swagger.json]: done

File: server_api_swagger.json [hunk 2]
```diff
@@ -1755,6 +1755,33 @@
...
+    "/contentpacks/installed/upload": {
+      "post": {
+        "description": "Upload an pack to marketplace in Demisto",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Upload an pack as zip",
```
Comment (glicht) [inline server_api_swagger.json:1767]:
```suggestion
        "summary": "Upload a Pack as zip file. The zip file maybe a single Pack or a zip containing multiple zipped Packs (a zip of zips)",
```

File: server_api_swagger.json [hunk 3]
```diff
@@ -1755,6 +1755,33 @@
         }
       }
     },
+    "/contentpacks/installed/upload": {
+      "post": {
+        "description": "Upload an pack to marketplace in Demisto",
```
Comment (glicht) [inline server_api_swagger.json:1760]:
```suggestion
        "description": "Upload a Pack to the marketplace in the Server. Can be used to upload a Pack for an offline scenario or a Pack that hasn't been released.",
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,8 +3,13 @@
 [PyPI History][1]
 
 [1]: https://pypi.org/project/demisto-py/#history
+
+
+## 3.0.3
+* Support `upload_content_packs` to upload zipped pack to marketplace.
```
Comment (glicht) [inline CHANGELOG.md:9]: I think you can unite this with 3.0.2 as it looks like 3.0.2 wasn't released yet.

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,12 +4,9 @@
 
 [1]: https://pypi.org/project/demisto-py/#history
 
-
-## 3.0.3
-* Support `upload_content_packs` to upload zipped pack to marketplace.
-
 ## 3.0.2
 * Support `upload_report` to upload a new report/ update existing report.
+* Support `upload_content_packs` to upload zipped pack to marketplace.
```
Comment (glicht) [inline CHANGELOG.md:9]:
```suggestion
* Support `upload_content_packs` to upload zipped packs to marketplace.
```

**Review by glicht — APPROVED** [review]

File: server_api_swagger.json (hunk 2 above)
Comment (ilappe) [reply server_api_swagger.json]: done

**Pull request merged**